wpilibc/shared/include/Commands/TimedCommand.h

```diff
@@ -1,20 +1,21 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2011-2016. All Rights Reserved.                        */
+/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
 #pragma once
 
+#include <string>
```

Comment:
Per our style guide, there should be an empty line between header types. In this case, between C++ system includes and user includes.